scim-server/src/main/java/org/apache/directory/scim/server/configuration/ServerConfiguration.java

```diff
@@ -86,15 +86,15 @@ public SupportedConfiguration getChangePasswordConfiguration() {
   public BulkConfiguration getBulkConfiguration() {
     BulkConfiguration bulkConfiguration = new BulkConfiguration();
 
-    bulkConfiguration.setSupported(supportsBulk);
+    bulkConfiguration.setSupported(isSupportsBulk());
     bulkConfiguration.setMaxOperations(bulkMaxOperations);
     bulkConfiguration.setMaxPayloadSize(bulkMaxPayloadSize);
```

Comment:
Nice suggestion!
To keep this class constant, can you use the getters elsewhere in this class?

Reply:
Yes, you are right, I have changed getter usage in all possible cases.